apps/application-system/api/src/environments/environment.interface.ts

```diff
@@ -3,7 +3,7 @@ import { TemplateAPIConfig } from '@island.is/application/template-api-modules'
 import { ApplicationConfig } from '../app/modules/application/application.configuration'
 import { FileStorageConfig } from '@island.is/file-storage'
 import { SigningServiceOptions } from '@island.is/dokobit-signing'
-import { SyslumennClientConfig } from '@island.is/api/domains/syslumenn'
+import { AuditOptions } from '@island.is/nest/audit'
```

Comment:
This config is not necesary anymore? `import { SyslumennClientConfig } from '@island.is/api/domains/syslumenn'`

Reply:
It’s not used. It’s part of the template api options instead.